libraries/aws_efs_file_system.rb

```diff
@@ -0,0 +1,80 @@
+# frozen_string_literal: true
+
+require 'aws_backend'
+
+class AwsEfsFileSystem < AwsResourceBase
+  name 'aws_efs_file_system'
+  desc 'Verifies settings for an EFS File System'
+
+  example "
+    describe aws_efs_file_system('fs-12345678') do
```

Comment:
Is this a valid example? It seems based on the checks that the `file_system_id: ` field needs to be added for this to work. 
Think there's a similar example in the README.

Reply:
Yes, it is a valid example.
The `file_system_id` is auto-assigned after the resource has been created.

I'll update the property definition to make it clearer.

Reply:
Add a note explaining default behavior for string parameters.
https://github.com/inspec/inspec-aws/pull/198/commits/6ad5abc1f12fa04e0284f46e9226a96ae0b1ebe7